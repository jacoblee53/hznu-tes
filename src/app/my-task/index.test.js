import React from 'react'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'
import injects from '../../inject/index'
import MyTask from './index'

let wrappedComponent = <MyTask {...injects} />

let wrapper

describe('src/app/my-task/index.test.js', () => {
  it('should render correctly', () => {
    wrapper = mount(wrappedComponent)
    expect(wrapper.html()).toBeTruthy()
  })

  it('should matchSnapshot', () => {
    const tree = renderer.create(wrappedComponent)
    expect(tree).toMatchSnapshot()
  })
})
