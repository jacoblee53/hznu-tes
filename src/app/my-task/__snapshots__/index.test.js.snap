// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/app/my-task/index.test.js should matchSnapshot 1`] = `
<div
  className="ant-card"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        我的任务
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <div
      className="ant-list ant-list-sm ant-list-split ant-list-something-after-last-item"
    >
      <div
        className="ant-spin-nested-loading"
      >
        <div
          className="ant-spin-container"
        >
          <div
            className="ant-list-empty-text"
          >
            <div
              className="ant-empty ant-empty-normal"
            >
              <div
                className="ant-empty-image"
              >
                <img
                  alt="No Data"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNjQiIGhlaWdodD0iNDEiIHZpZXdCb3g9IjAgMCA2NCA0MSIgIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CiAgPGcgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoMCAxKSIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIj4KICAgIDxlbGxpcHNlIGZpbGw9IiNGNUY1RjUiIGN4PSIzMiIgY3k9IjMzIiByeD0iMzIiIHJ5PSI3Ii8+CiAgICA8ZyBmaWxsLXJ1bGU9Im5vbnplcm8iIHN0cm9rZT0iI0Q5RDlEOSI+CiAgICAgIDxwYXRoIGQ9Ik01NSAxMi43Nkw0NC44NTQgMS4yNThDNDQuMzY3LjQ3NCA0My42NTYgMCA0Mi45MDcgMEgyMS4wOTNjLS43NDkgMC0xLjQ2LjQ3NC0xLjk0NyAxLjI1N0w5IDEyLjc2MVYyMmg0NnYtOS4yNHoiLz4KICAgICAgPHBhdGggZD0iTTQxLjYxMyAxNS45MzFjMC0xLjYwNS45OTQtMi45MyAyLjIyNy0yLjkzMUg1NXYxOC4xMzdDNTUgMzMuMjYgNTMuNjggMzUgNTIuMDUgMzVoLTQwLjFDMTAuMzIgMzUgOSAzMy4yNTkgOSAzMS4xMzdWMTNoMTEuMTZjMS4yMzMgMCAyLjIyNyAxLjMyMyAyLjIyNyAyLjkyOHYuMDIyYzAgMS42MDUgMS4wMDUgMi45MDEgMi4yMzcgMi45MDFoMTQuNzUyYzEuMjMyIDAgMi4yMzctMS4zMDggMi4yMzctMi45MTN2LS4wMDd6IiBmaWxsPSIjRkFGQUZBIi8+CiAgICA8L2c+CiAgPC9nPgo8L3N2Zz4K"
                />
              </div>
              <p
                className="ant-empty-description"
              >
                No Data
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-list-footer"
      >
        <div />
      </div>
      <div
        className="ant-list-pagination"
      >
        <ul
          className="ant-pagination "
          style={Object {}}
          unselectable="unselectable"
        >
          <li
            aria-disabled={true}
            className="ant-pagination-disabled ant-pagination-prev"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex={null}
            title="Previous Page"
          >
            <a
              className="ant-pagination-item-link"
            >
              <i
                aria-label="icon: left"
                className="anticon anticon-left"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="left"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 0 0 0 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                  />
                </svg>
              </i>
            </a>
          </li>
          <li
            className="ant-pagination-item ant-pagination-item-0 ant-pagination-disabled ant-pagination-item-disabled"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex="0"
            title={0}
          >
            <a>
              0
            </a>
          </li>
          <li
            aria-disabled={true}
            className="ant-pagination-disabled ant-pagination-next"
            onClick={[Function]}
            onKeyPress={[Function]}
            tabIndex={null}
            title="Next Page"
          >
            <a
              className="ant-pagination-item-link"
            >
              <i
                aria-label="icon: right"
                className="anticon anticon-right"
              >
                <svg
                  aria-hidden="true"
                  className=""
                  data-icon="right"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M765.7 486.8L314.9 134.7A7.97 7.97 0 0 0 302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 0 0 0-50.4z"
                  />
                </svg>
              </i>
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
